sibylapp/view/feature_contribution.py

```diff
@@ -2,10 +2,31 @@
 
 from sibylapp.compute import contributions
 from sibylapp.compute.context import get_term
+from sibylapp.config import PREDICTION_TYPE, PredType, negativeTerm, positiveTerm
 from sibylapp.view.utils import filtering, helpers
+from sibylapp.view.utils.helpers import NEG_EM, POS_EM
+
+
+def legend():
+    modelPred = get_term("Prediction")
+    posChange = ""
+    negChange = ""
+    separator = "           "
+    # TODO: add detail for categorical predictions after we figure this out
+    if PREDICTION_TYPE == PredType.NUMERIC:
+        posChange = " Increase in"
+        negChange = " Decrease in"
+    elif PREDICTION_TYPE == PredType.BOOLEAN:
+        posChange = " towards " + positiveTerm + " as"
+        negChange = " towards " + negativeTerm + " as"
+    st.write(
+        (POS_EM + posChange + " " + modelPred) + separator + (NEG_EM + negChange + " " + modelPred)
```

Comment:
I like you including the prediction name - I think that wording is very clear. To make this a bit faster to read, I think we should bold the "posChange/negChange" word, so it would look like:
towards **Failure** as prediction outcome
If you want to keep consistent cases, you can use `get_term("Prediction", l=True)` to get the prediction term in lowercase